build/patches/Disable-text-fragments-by-default.patch

```diff
@@ -127,58 +126,10 @@ diff --git a/third_party/blink/renderer/core/dom/document.cc b/third_party/blink
  
    url_ = new_url;
    UpdateBaseURL();
-diff --git a/third_party/blink/renderer/core/fragment_directive/text_fragment_anchor_metrics_test.cc b/third_party/blink/renderer/core/fragment_directive/text_fragment_anchor_metrics_test.cc
```

Comment:
Code removed in Deprecate unused scroll-to-text micro metrics | https://chromium-review.googlesource.com/c/chromium/src/+/3629958